backend/main.py

```
@@ -208,6 +249,18 @@ async def get_current_user_profile(current_user: User = Depends(get_current_user
     """Get current user profile"""
     return current_user
 
+@app.post("/test-login")
+async def test_login(user: UserLogin, db: Session = Depends(get_db)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)